micrometer-core/gradle.lockfile

```diff
@@ -104,8 +104,8 @@ org.apache.kafka:kafka-clients:2.5.0=compileClasspath,runtimeClasspath,testCompi
 org.apache.kafka:kafka-streams:2.5.0=compileClasspath,runtimeClasspath,testCompileClasspath,testRuntimeClasspath
 org.apache.logging.log4j:log4j-api:2.13.2=compileClasspath,runtimeClasspath,testCompileClasspath,testRuntimeClasspath
 org.apache.logging.log4j:log4j-core:2.13.2=compileClasspath,runtimeClasspath,testCompileClasspath,testRuntimeClasspath
-org.apache.tomcat.embed:tomcat-embed-core:8.5.54=compileClasspath,runtimeClasspath,testCompileClasspath,testRuntimeClasspath
-org.apache.tomcat:tomcat-annotations-api:8.5.54=compileClasspath,runtimeClasspath,testCompileClasspath,testRuntimeClasspath
+org.apache.tomcat.embed:tomcat-embed-core:8.5.69=compileClasspath,runtimeClasspath,testCompileClasspath,testRuntimeClasspath
```

Comment:
Our Tomcat tests fail on Java 16 without this upgrade due to an InaccessibleObjectException.
https://tomcat.apache.org/tomcat-8.5-doc/changelog.html#Tomcat_8.5.62_(markt)